profile/README.md

```diff
@@ -0,0 +1,55 @@
+## Hi there, we are Fobnail Team 👋
+
+The Fobnail project provides the resources to create an axiomatically
+trustworthy device and a simple user interface to confirm the status of the
+platform. We aim to provide a reference architecture for building
+offline integrity measurement verifiers on a USB device and validators running
+in dynamically run measured environments (DLME).
+
+Our goal is to create a scalable, flexible implementation that will be the main
+building block of future (community) security solutions based on DRTM payloads.
+The owner of Fobnail can verify the reliability of the working system before
+performing any sensitive operation. Fobnail does not need an internet
+connection, which makes it immune to network stack attacks and remote
+infrastructure.
+
+[The Fobnail Token](https://fobnail.3mdeb.com/fobnail_token/) is a small
+open-source USB hardware reference project that allows a
+user/administrator/company to determine system integrity.
+
+The Fobnail project has the potential to scale up in the future. The Fobnail
+architecture can be adapted to devices using other buses than USB.
+A detailed description of the project can be found
+[here](https://fobnail.3mdeb.com/).
```

Comment:
The last sentence doesn't fit with the rest, it should be in separate paragraph, maybe together with the next one.

Reply:
Separated and connected with next sentence.